pkg/resolver/searchHelper.go

```diff
@@ -24,33 +23,61 @@ func getPropertyType(ctx context.Context, refresh bool) (map[string]string, erro
 	return propTypesCache, err
 }
 
-// Remove operator (<=, >=, !=, !, <, >, =) if any from values
-func getOperator(values []string) map[string][]string {
-	// Get the operator (/^<=|^>=|^!=|^!|^<|^>|^=/)
-	var operator string
-	// Replace any of these symbols with ""
-	replacer := strings.NewReplacer("<=", "",
-		">=", "",
-		"!=", "",
-		"!", "",
-		"<", "",
-		">", "",
-		"=", "")
-	operatorValue := map[string][]string{}
+// Extract operator (<=, >=, !=, !, <, >, =) if any from string
+func getOperatorFromString(value string) (string, string) {
+	klog.Info("value: ", value)
+	operator := "="
+	operand := value
+
+	prefixes := []string{"<=", ">=", "!=", "!", "<", ">", "="}
+	for _, prefix := range prefixes {
+		if cutString, yes := strings.CutPrefix(value, prefix); yes {
+			operator = prefix
+			operand = cutString
+			klog.Info("operator: ", operator)
+			klog.Info("operand: ", operand)
```

Comment:
Lower verbosity.  Consider combining all 3 logs in this function into a single log.